contracts/interfaces/modules/royalty/IRoyaltyModule.sol

```diff
@@ -30,9 +30,18 @@ interface IRoyaltyModule is IModule {
     /// @param amount The amount paid
     event LicenseMintingFeePaid(address receiverIpId, address payerAddress, address token, uint256 amount);
 
+    /// @notice Sets the license registry
+    /// @dev Enforced to be only callable by the protocol admin
+    /// @param licensing The address of the license registry
+    /// @param dispute The address of the dispute module
+    function setLicensingAndDisputeModules(address licensing, address dispute) external;
```

Comment:
small nit, other modules, and registries have one setter function per address (e.g. setLicensing, setDispute, etc.). FYI to maintain uniformity in the codebase.

Reply:
https://github.com/storyprotocol/protocol-core-v1/pull/35/commits/213e2b7ad4314abc59b273e79d364c384a80499f